common-ui-compose/src/main/java/app/tivi/common/compose/Carousel.kt

```diff
@@ -40,10 +40,12 @@ fun <T> Carousel(
 
     LazyRow(
         modifier = modifier,
-        contentPadding = contentPadding.copy(
-            start = (contentPadding.start - halfSpacing).coerceAtLeast(0.dp),
-            end = (contentPadding.end - halfSpacing).coerceAtLeast(0.dp)
-        ),
+        contentPadding = contentPadding,
+        // FIXME
+//        .copy(
```

Comment:
Are you leaving this as a _TODO_ or fixing it for this version?

Reply:
Fixed.